### ISSUE TITLE
Enable KVM acceleration support 

### ISSUE THREAD

**Issue by DoktorCranium** [DC-DeepComputing/DC-ROMA_Pad_II_RV-P1B#11]
Enable KVM acceleration support on the Ubuntu 24.04 kernel as shipped on the DC-ROMA Pad II 

Linux dcroma2 6.6.36 #1 SMP PREEMPT Sat Sep 14 05:50:14 UTC 2024 riscv64 riscv64 riscv64 GNU/Linux

More information on this 
https://github.com/kvm-riscv/howto/wiki

This would be great to enable native virtualization of other riscv systems on the tablet  (LInux, FreeBSD, NetBSD)

**Comment by Haui1112** [comment]
From a cursory overview it seems like building a custom kernel is all that is needed here. have you tried building it and exchanging in and its initramfs on a copy of the debian sd card? Maybe I'm missing something.

**Comment by DoktorCranium** [comment]
Yes - you can always compile the kernel given the sources here https://github.com/DC-DeepComputing/DC-linux - question is, are the PowerVR GPU sources included here as well ? I can try experiment with the Debian SD card custom kernel - but eventually it would be nice to have this enabled by default in the stock image (after all the DC ROMA II is a DEV machine)

**Comment by sandiecao-dc** [comment]
DC-linux repo is for fml13v01 board, not for PAD II.  And the powerVR GPU source isn't included in DC-linux repo.
